tests/e2e/test_running_cluster_monitoring_with_persistent_storage.py

```diff
@@ -0,0 +1,77 @@
+import logging
+
+import pytest
+
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.framework.testlib import tier1, E2ETest
+from ocs_ci.ocs.resources.pvc import delete_pvcs
+from ocs_ci.ocs.monitoring import (
+    create_configmap_cluster_monitoring_pod,
+    validate_pvc_created_and_bound_on_monitoring_pods,
+    validate_pvc_are_mounted_on_monitoring_pods,
+    validate_monitoring_pods_are_respinned_and_running_state,
+    get_list_pvc_objs_created_on_monitoring_pods
+)
+from tests.fixtures import (
+    create_rbd_storageclass, create_ceph_block_pool,
+    create_rbd_secret
+)
+
+logger = logging.getLogger(__name__)
+ocp = OCP('v1', 'ConfigMap', 'openshift-monitoring')
+
+
+@pytest.fixture()
+def test_fixture(request):
+    """
+    Setup and teardown
+    """
+    self = request.node.cls
+
+    def finalizer():
+        teardown(self)
+    request.addfinalizer(finalizer)
+
+
+def teardown(self):
```

Comment:
@Akarsha-rai what will happen to monitoring pods after teardown if we delete the configuration abruptly? I think they'll crash. is there a  better way?